src/constants/queries.ts

```diff
@@ -169,21 +185,44 @@ export const BreakdownQueryMapODF = (label: string, queryType: string) => {
 export const UTILIZATION_QUERY_ODF = (label: string, func: string) => {
   switch (func) {
     case StorageDashboardQuery.UTILIZATION_CAPACITY_QUERY:
+      // return [
+      //   {
+      //     query: FlASHSYSTEM_QUERIES(
+      //       label,
+      //       StorageDashboardQuery.TotalUsedCapacity
+      //     ),
+      //     desc: "Used",
+      //   },
+      //   {
+      //     query: FlASHSYSTEM_QUERIES(
+      //       label,
+      //       StorageDashboardQuery.TotalFreeCapacity
+      //     ),
+      //     desc: "Available",
+      //   },
+      // ];
```

Comment:
I think we can remove these commented lines, they are not relevant anymore.

Reply:
Done